keyless/pepper/service/src/jwk.rs

```diff
@@ -50,17 +51,24 @@ pub fn start_jwk_refresh_loop(issuer: &str, jwk_url: &str, refresh_interval: Dur
     let jwk_url = jwk_url.to_string();
     let _handle = tokio::spawn(async move {
         loop {
-            match fetch_jwks(jwk_url.as_str()).await {
+            let timer = Instant::now();
+            let fetch_result = fetch_jwks(jwk_url.as_str()).await;
+            let fetch_time = timer.elapsed();
+            let succeeded = fetch_result.is_ok();
+            JWK_FETCH_SECONDS
+                .with_label_values(&[issuer.as_str(), succeeded.to_string().as_str()])
```

Comment:
Do you wanna log other things, like JWK size/type?

Reply:
it's very frequent but not too useful, so not logging them for now.